Evaluation part of integration test fails

### What happened?

Evaluation is currently disabled in the integration test:

https://github.com/ecmwf/WeatherGenerator/blob/3d27b6a0fe07908d6d2198327d703899ada091bd/integration_tests/small_multi_stream_test.py#L71

When it is enabled, the integration test fails with:

```
FAILEDend fixture


====================================================== FAILURES =======================================================
__________________________________ test_train_multi_stream[test_multi_stream_e4c51] ___________________________________

setup = None, test_run_id = 'test_multi_stream_e4c51'

    @pytest.mark.parametrize("test_run_id", ["test_multi_stream_" + commit_hash])
    def test_train_multi_stream(setup, test_run_id):
        """Test training with multiple streams including gridded and observation data."""
        logger.info(f"test_train_multi_stream with run_id {test_run_id} {WEATHERGEN_HOME}")
    
        train_with_args(
            f"--config={WEATHERGEN_HOME}/integration_tests/small_multi_stream.yaml".split()
            + [
                "--run_id",
                test_run_id,
            ],
            f"{WEATHERGEN_HOME}/integration_tests/streams_multi/",
        )
    
        infer_multi_stream(test_run_id)
>       evaluate_multi_stream_results(test_run_id)

integration_tests/small_multi_stream_test.py:71: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
integration_tests/small_multi_stream_test.py:158: in evaluate_multi_stream_results
    evaluate_from_config(cfg, None, None)
packages/evaluate/src/weathergen/evaluate/run_evaluation.py:305: in evaluate_from_config
    type_, run, run_id, private_paths, cfg.evaluation.regions, cfg.evaluation.metrics
.venv/lib/python3.12/site-packages/omegaconf/dictconfig.py:355: in __getattr__
    self._format_and_raise(
.venv/lib/python3.12/site-packages/omegaconf/base.py:231: in _format_and_raise
    format_and_raise(
.venv/lib/python3.12/site-packages/omegaconf/_utils.py:899: in format_and_raise
    _raise(ex, cause)
.venv/lib/python3.12/site-packages/omegaconf/_utils.py:797: in _raise
    raise ex.with_traceback(sys.exc_info()[2])  # set env var OC_CAUSE=1 for full trace
.venv/lib/python3.12/site-packages/omegaconf/dictconfig.py:351: in __getattr__
    return self._get_impl(
.venv/lib/python3.12/site-packages/omegaconf/dictconfig.py:442: in _get_impl
    node = self._get_child(
.venv/lib/python3.12/site-packages/omegaconf/basecontainer.py:73: in _get_child
    child = self._get_node(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = {'metrics': ['rmse', 'l1', 'mse'], 'verbose': True, 'summary_plots': True, 'summary_dir': './plots/', 'print_summary': True}
key = 'regions', validate_access = True, validate_key = False, throw_on_missing_value = False
throw_on_missing_key = True

    def _get_node(
        self,
        key: DictKeyType,
        validate_access: bool = True,
        validate_key: bool = True,
        throw_on_missing_value: bool = False,
        throw_on_missing_key: bool = False,
    ) -> Optional[Node]:
        try:
            key = self._validate_and_normalize_key(key)
        except KeyValidationError:
            if validate_access and validate_key:
                raise
            else:
                if throw_on_missing_key:
                    raise ConfigAttributeError
                else:
                    return None
    
        if validate_access:
            self._validate_get(key)
    
        value: Optional[Node] = self.__dict__["_content"].get(key)
        if value is None:
            if throw_on_missing_key:
>               raise ConfigKeyError(f"Missing key {key!s}")
E               omegaconf.errors.ConfigAttributeError: Missing key regions
E                   full_key: evaluation.regions
E                   object_type=dict

.venv/lib/python3.12/site-packages/omegaconf/dictconfig.py:480: ConfigAttributeError
```

CC @iluise 



### What are the steps to reproduce the bug?

Run integration test with https://github.com/ecmwf/WeatherGenerator/blob/3d27b6a0fe07908d6d2198327d703899ada091bd/integration_tests/small_multi_stream_test.py#L71 commented in

### Hedgedoc link to logs and more information. This ticket is public, do not attach files directly.

_No response_

--- a/integration_tests/small1_test.py
+++ b/integration_tests/small1_test.py
@@ -15,8 +15,8 @@ from pathlib import Path
 
 import omegaconf
 import pytest
-
 from weathergen.evaluate.run_evaluation import evaluate_from_config
+
 from weathergen.run_train import inference_from_args, train_with_args
 from weathergen.utils.metrics import get_train_metrics_path
 
@@ -105,6 +105,7 @@ def evaluate_results(run_id):
                 "dpi_val": 300,
             },
             "evaluation": {
+                "regions": ["global"],
                 "metrics": ["rmse", "l1", "mse"],
                 "verbose": True,
                 "summary_plots": True,

--- a/integration_tests/small_multi_stream_test.py
+++ b/integration_tests/small_multi_stream_test.py
@@ -23,8 +23,8 @@ from pathlib import Path
 
 import omegaconf
 import pytest
-
 from weathergen.evaluate.run_evaluation import evaluate_from_config
+
 from weathergen.run_train import inference_from_args, train_with_args
 from weathergen.utils.metrics import get_train_metrics_path
 
@@ -105,6 +105,7 @@ def evaluate_multi_stream_results(run_id):
                 "dpi_val": 300,
             },
             "evaluation": {
+                "regions": ["global"],
                 "metrics": ["rmse", "l1", "mse"],
                 "verbose": True,
                 "summary_plots": True,

